draft-ietf-rats-corim.md

```diff
@@ -481,6 +481,14 @@ Described in {{sec-common-validity}}.
 {::include cddl/unprotected-corim-header-map.cddl}
 ~~~
 
+## Signer authority of securely conveyed unsigned CoRIM {#sec-conveyed-signer}
+
+An unsigned (#6.501-tagged) CoRIM may be contained in a larger signed document.
```

Comment:
"document" usually means "specification". Maybe "payload" instead of "document"?

Reply:
Accepted Ned's changes. Thanks.